tiledb/sm/subarray/subarray.cc

```diff
@@ -555,6 +837,95 @@ Status Subarray::get_est_result_size(
   return Status::Ok();
 }
 
+Status Subarray::get_est_result_size(
+    const char* name, uint64_t* size, StorageManager* storage_manager) {
+  QueryType type;
+  // Note: various items below expect array open, get_query_type() providing
+  // that audit.
+  RETURN_NOT_OK(array_->get_query_type(&type));
+
+  if (type == QueryType::WRITE)
```

Comment:
If you have a minute, could we please double check that nothing has changed in the changes we do in query since this was done.

Reply:
functionality compared earlier vs more recent, 
did not see any changes in query.cc that would indicates changes required in this area of subarray.cc